README.rst

```diff
@@ -100,21 +100,27 @@ refer to `compiling.rst`_ or build the documentation, cf. below.
 **Linking libdrizzle-redux**::
 
 Ensure the library is in your library and include paths. For releases prior to
-version ``v6.0.2`` linking your app against libdrizzle-redux requires the flag
-``-ldrizzle-redux``::
+version ``v6.0.3`` linking your app against libdrizzle-redux requires the flag
+``-ldrizzle-redux6``::
```

Comment:
I would just say `v6.x.x`, as those point release are just broken and it creates confusion. But I'm not big on that point.